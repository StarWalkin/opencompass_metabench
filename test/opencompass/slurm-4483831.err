  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:02<00:00,  2.10s/it]100%|██████████| 1/1 [00:02<00:00,  2.10s/it]
  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:07<00:00,  7.95s/it]100%|██████████| 1/1 [00:07<00:00,  7.95s/it]
