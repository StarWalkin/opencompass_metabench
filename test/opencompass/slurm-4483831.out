/u/zzhang27/.conda/envs/opencompass/bin/python3
08/15 22:37:22 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/15 22:37:22 - OpenCompass - INFO - Loading hf_qwen2_0_5b_instruct: configs/models/qwen/hf_qwen2_0_5b_instruct.py
08/15 22:37:22 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/15 22:37:22 - OpenCompass - INFO - Current exp folder: outputs/default/20240815_223722
08/15 22:37:22 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/15 22:37:22 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLInfer[qwen2-0.5b-instruct-hf/gsm8k] on GPU 0
08/15 22:37:25 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/local.py - _launch - 228 - task OpenICLInfer[qwen2-0.5b-instruct-hf/gsm8k] fail, see
outputs/default/20240815_223722/logs/infer/qwen2-0.5b-instruct-hf/gsm8k.out
08/15 22:37:25 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/base.py - summarize - 64 - OpenICLInfer[qwen2-0.5b-instruct-hf/gsm8k] failed with code 1
08/15 22:37:25 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLEval[qwen2-0.5b-instruct-hf/gsm8k] on CPU 
dataset    version    metric    mode    qwen2-0.5b-instruct-hf
---------  ---------  --------  ------  ------------------------
gsm8k      -          -         -       -
08/15 22:37:33 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240815_223722/summary/summary_20240815_223722.txt
08/15 22:37:33 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240815_223722/summary/summary_20240815_223722.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
